srcpkgs/gomuks/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gomuks'
+pkgname=gomuks
+version=0.2.2
+revision=1
+build_style=go
+go_import_path="maunium.net/go/gomuks"
+makedepends="olm-devel"
+short_desc="Terminal based Matrix client written in Go"
+maintainer="FollieHiyuki <folliekazetani@protonmail.com>"
+license="AGPL-3.0-or-later"
+homepage="https://github.com/tulir/gomuks"
```

Comment:
Should be https://maunium.net/go/gomuks/

Reply:
Changed.